packages/compass-aggregations/src/modules/clone-pipeline.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Clone pipeline action name.
+ */
+export const CLONE_PIPELINE = 'aggregations/CLONE_PIPELINE';
+
+/**
+ * The clone pipeline action.
+ */
+export const clonePipeline = () => ({
```

Comment:
We don't really need this action as far as I can see, this can be handled by a single event, especially taking into account that spawning multiple events from a single user action (and not from e.g. thunk action) is considered a bad practice, but I am still trying to keep changes to the minimum here